deploy/Vagrantfile

```diff
@@ -32,7 +32,7 @@ Vagrant.configure(2) do |config|
     fs_config.vm.provider 'virtualbox' do |v|
       unless File.exist?(disk_file_1)
         v.customize ['createhd', '--filename', disk_file_1, '--size', 2048, '--variant', 'fixed']
-        v.customize ['storagectl', :id, '--name', 'SATA Controller', '--add', 'sata', '--portcount', 1]
+#        v.customize ['storagectl', :id, '--name', 'SATA Controller', '--add', 'sata', '--portcount', 1]
```

Comment:
@olivermel Can you just add a comment here that says: "TODO: Detect if SATA controller exists, and only add it if needed."